src/main/java/uk/gov/hmcts/reform/pip/model/publication/ListType.java

```diff
@@ -63,7 +63,19 @@ public enum ListType {
     PHT_WEEKLY_HEARING_LIST(NATIONAL, CFT_IDAM, ALL_VERIFIED_THIRD_PARTY_CFT_ROLES),
     GRC_WEEKLY_HEARING_LIST(NATIONAL, CFT_IDAM, ALL_VERIFIED_THIRD_PARTY_CFT_ROLES),
     WPAFCC_WEEKLY_HEARING_LIST(NATIONAL, CFT_IDAM, ALL_VERIFIED_THIRD_PARTY_CFT_ROLES),
-    UT_IAC_JUDICIAL_REVIEW_DAILY_HEARING_LIST(NATIONAL, CFT_IDAM, ALL_VERIFIED_THIRD_PARTY_CFT_ROLES),
+    UT_IAC_JR_LONDON_DAILY_HEARING_LIST(NATIONAL, CFT_IDAM, ALL_VERIFIED_THIRD_PARTY_CFT_ROLES),
```

Comment:
There is no Explicit friendly name like the other JR lists, so it is going to be converted to UT IAC JR London Daily Hearing List. The other ones are like "Upper Tribunal (Immigration and Asylum) Chamber - Judicial Review: XXX Daily Hearing List". Make them consistent

Reply:
done